### DISCUSSION TITLE
Feature Request: allow to configure Lambda stack and application assignment

### DISCUSSION THREAD

**Discussion by cforce** [artilleryio/artillery#1795] [Artillery]
I recognized that the lambda provider tasks is created without an stack and application assignment. It would be of advance to mange the test driver task to be created in the same (existing) cloud formation stack and application as the task under test/load.

**Comment by hassy** [comment]
hi @cforce - could you expand a little bit on how this would work? At the moment, all resources needed to run the test are created on-the-fly, and then deleted after the test is finished. The two exceptions are an IAM role and an S3 bucket. Would it be useful to have those resources managed by CloudFormation stack?

**Comment by cforce** [comment]
Its rather simple and mostly about  "organizing"  resources in the same box. 
https://docs.aws.amazon.com/lambda/latest/dg/deploying-lambda-apps.html
https://docs.aws.amazon.com/lambda/latest/dg/applications-console.html

However i didged  a bit deeper and it doesn't make sense as we have two entities - one is the application under load which is managed by one application / formation  template which bundle and deploys all resources togehter in one transaction. The artillery task is standalone and the whole resource provisioning for it as well. IT would be at least be handy  to have full control on the naming of that generated/deployed artillery resources - so its easier  to locate all stuff from one root easily.
